tools/cli/internal/openapi/filter/operations.go

```diff
@@ -36,36 +35,9 @@ func (f *OperationsFilter) Apply() error {
 		for _, operation := range pathItem.Operations() {
 			if operation.Extensions != nil {
 				delete(operation.Extensions, "x-xgen-owner-team")
+				delete(operation.Extensions, "x-xgen-changelog")
```

Comment:
Do we want to remove also `x-xgen-IPA-exception` ?

Reply:
it's done by another filter

Reply:
I am confused 🤔 Isn't the goal here to remove all the custom extensions from an openapi spec so that integrations (AtlasCLI) can use it without worry of which extension they should parse?

Reply:
sorry the exception is being filtered out as well as you suggested, but in another filter file as the x-xgen-IPA-exception is in many layers of the openapi spec!